### ISSUE TITLE
All chapters are hidden even the unread ones[BUG]

### ISSUE THREAD

**Issue by lol480phuh** [Glagan/SyncDex#52]
**Describe the bug**

On my `follows` page all chapters are hidden, even the new chapters, also they're not highlighted unless I click the `Show Hidden` which shows everything including older chapters, it was fine in `0.3.1` haven't checked `0.3.2` since I didn't read anything yesterday, but I received the `0.3.3` this morning and went to the `Follows` page and that problem occurred.

**To Reproduce**

1. Go to https://mangadex.org/follows/
2. Chapters appear as usual for a brief moment (new and old chapters alike, highlighted with their respective colors)
3. All chapters disappear like what happens you're up to date with all the chapters
4. Click `Show Hidden` and all the chapters including the older ones appear.

**Expected behavior (optional)**

Only new chapters appear and they would be highlighted.

**Screenshots**
![Screenshot_38](https://user-images.githubusercontent.com/52804334/111148817-61f08380-858c-11eb-8d9a-06de55e99bb5.png)
![Screenshot_39](https://user-images.githubusercontent.com/52804334/111148833-65840a80-858c-11eb-9fc8-2381ec5d8960.png)
![Screenshot_41](https://user-images.githubusercontent.com/52804334/111148895-7af93480-858c-11eb-8e22-1558c9a475cd.png)

And click `Show Hidden` to get this:

![Screenshot_42](https://user-images.githubusercontent.com/52804334/111148913-7e8cbb80-858c-11eb-9f7f-0056476a7ddf.png)

**Browser**
Microsoft Edge (Chromium) Version 88.0.705.53 (Official build) (64-bit)

**Options and logs**

```
{"logs":[{"d":1611784908892,"msg":"Installation version 0.2.7"},{"d":1611982174112,"msg":"Updating from version 0.2.7 to 0.2.8"},{"d":1612104915509,"msg":"Updated from version 0.2.8 to 0.2.9"},{"d":1612121643537,"msg":"Updated from version 0.2.9 to 0.2.10"},{"d":1612191086897,"msg":"Updated from version 0.2.10 to 0.2.11"},{"d":1612386743786,"msg":"Error on request [https://mangadex.org/api/v2/user/me/manga/44146]: SyntaxError: Unexpected token < in JSON at position 0>> SyntaxError: Unexpected token < in JSON at position 0"},{"d":1612896819970,"msg":"Updated from version 0.2.11 to 0.2.12"},{"d":1613334958960,"msg":"Error on request [https://mangadex.org/api/v2/user/me/manga/44]: SyntaxError: Unexpected token < in JSON at position 0>> SyntaxError: Unexpected token < in JSON at position 0"},{"d":1613335698197,"msg":"Error on request [https://mangadex.org/api/v2/user/me/manga/42620]: SyntaxError: Unexpected token < in JSON at position 0>> SyntaxError: Unexpected token < in JSON at position 0"},{"d":1613656481091,"msg":"Error on request [https://graphql.anilist.co]: SyntaxError: Unexpected token < in JSON at position 0>> SyntaxError: Unexpected token < in JSON at position 0"},{"d":1613656497450,"msg":"Error on request [https://graphql.anilist.co]: SyntaxError: Unexpected token < in JSON at position 0>> SyntaxError: Unexpected token < in JSON at position 0"},{"d":1613656685743,"msg":"Error on request [https://graphql.anilist.co]: SyntaxError: Unexpected token < in JSON at position 0>> SyntaxError: Unexpected token < in JSON at position 0"},{"d":1613656723131,"msg":"Error on request [https://graphql.anilist.co]: SyntaxError: Unexpected token < in JSON at position 0>> SyntaxError: Unexpected token < in JSON at position 0"},{"d":1613656753028,"msg":"Error on request [https://graphql.anilist.co]: SyntaxError: Unexpected token < in JSON at position 0>> SyntaxError: Unexpected token < in JSON at position 0"},{"d":1613658243168,"msg":"Error on request [https://graphql.anilist.co]: SyntaxError: Unexpected token < in JSON at position 0>> SyntaxError: Unexpected token < in JSON at position 0"},{"d":1613686547713,"msg":"Error on request [https://graphql.anilist.co]: SyntaxError: Unexpected token < in JSON at position 0>> SyntaxError: Unexpected token < in JSON at position 0"},{"d":1613687142455,"msg":"Error on request [https://graphql.anilist.co]: SyntaxError: Unexpected token < in JSON at position 0>> SyntaxError: Unexpected token < in JSON at position 0"},{"d":1613687612397,"msg":"Error on request [https://graphql.anilist.co]: SyntaxError: Unexpected token < in JSON at position 0>> SyntaxError: Unexpected token < in JSON at position 0"},{"d":1614195903912,"msg":"Updated from version 0.2.12 to 0.2.14"},{"d":1614196005958,"msg":"Error on request [https://myanimelist.net/ownlist/manga/123694/edit?hideLayout]: TypeError: Failed to fetch>> TypeError: Failed to fetch"},{"d":1614272138463,"msg":"Updated from version 0.2.14 to 0.2.15"},{"d":1615562347889,"msg":"Updated from version 0.2.15 to 0.3.0"},{"d":1615579187298,"msg":"Applied patch version 0.3.1"},{"d":1615579187308,"msg":"Updated from version 0.3.0 to 0.3.1"},{"d":1615726873673,"msg":"Updated from version 0.3.1 to 0.3.2"},{"d":1615802567752,"msg":"Updated from version 0.3.2 to 0.3.3"}],"options":{"acceptLowScore":false,"autoSync":true,"biggerHistory":true,"chapterStatus":false,"chaptersSaved":400,"checkOnStartup":false,"checkOnStartupCooldown":30,"checkOnStartupMainOnly":true,"colors":{"higherChapter":"transparent","highlights":["rgba(82, 190, 90, 0.6)","rgba(107, 177, 95, 0.6)","rgba(55, 168, 61, 0.6)"],"lowerChapter":"rgba(180, 102, 75, 0.5)","nextChapter":"rgba(104, 115, 251, 0.4)","openedChapter":"rgba(28, 135, 141, 0.4)"},"confirmChapter":false,"displaySyncStart":false,"errorDuration":4000,"errorNotifications":true,"favoriteLanguage":"all","groupTitlesInLists":true,"hideHigher":false,"hideLast":true,"hideLower":true,"highlight":true,"iconsSilentAfterSync":false,"infoDuration":4000,"linkToServices":true,"logLevel":0,"mdUpdateSyncDex":true,"noReloadStatus":true,"notifications":true,"originalThumbnail":false,"overviewMainOnly":true,"progressInThumbnail":true,"saveOnLastPage":false,"saveOnlyHigher":true,"saveOnlyNext":false,"saveOpenedChapters":true,"separateLanguages":true,"services":["mal","al"],"silentUpdate":false,"subVersion":3,"successDuration":4000,"thumbnail":true,"thumbnailMaxHeight":80,"tokens":{"anilistToken":"set"},"updateMD":true,"updateMDProgress":true,"updateOnlyInList":false,"useMochi":true,"version":0.3}}
```

**Comment by Jerrk** [comment]
this was fixed in 3ac54eac26f8ad9ad6aac4d813c1932ab40b6ae7, so just update to 0.3.4 🙂

**Comment by Glagan** [comment]
Duplicate of #51.

This was fixed on  ``0.3.4`` as Jerrk said, it should also already be up for Chrome.  
You can manually check for updates by going to ``chrome://extensions/`` and clicking the **Update** button, but I don't know if there will be one, waiting until it automatically updates is another solution.

**Comment by lol480phuh** [comment]
Thank you for the quick fix, you guys are the best!
